### ISSUE TITLE
Carbon Update

### ISSUE THREAD

**Issue by HerbFargus** [wetriner/es-theme-gallery#1]
The carbon theme now has the gamelist on the left instead of the right. I'd submit a PR but I don't have the dummy image placemark

btw thanks for keeping this gallery updated! I've updated the theme wiki page from this repo:

https://github.com/RetroPie/RetroPie-Setup/wiki/Themes

seems two recent themes were also added: flat and fundamental

**Comment by wetriner** [comment]
Thanks for letting me know. I waited for flat and fundamental because emulationstation didn't have the video element support yet, so those themes just threw errors on retropie 4.1. I added those two and updated carbon. In addition, I went through the rest and updated changes that I saw in others.

The files I use are just for quick setup. PR's that don't use the same image and gamelist would be fine by me, but I'll add the image for reference. I currently only just follow changes made to esthemes. I'll should start following all of the theme repos for changes going forward.

![sample-image](https://cloud.githubusercontent.com/assets/17664918/24231487/0b360c78-0f29-11e7-9240-aaa8f5b1a36b.png)
